src/components/User/DetailsCard.tsx

```diff
@@ -6,28 +6,28 @@ interface dummyDataProps {
     email: string;
     status?: string;
     access?: string;
-  } | null;
+  };
```

Comment:
Type alias / Interfaces should start with uppercase letter.
Also, I see `UserType` interface has similar object shape. Reuse it here.
You may need to use [Omit utility type](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys).